build_js.sh

```diff
@@ -62,3 +62,7 @@ echo "Then open in browser:"
 echo "http://0.0.0.0:8000/build_js/utils/viewer/viewer.html?scene=skokloster-castle.glb"
 echo "Or:"
 echo "http://0.0.0.0:8000/build_js/esp/bindings_js/bindings.html?scene=skokloster-castle.glb"
+echo "To enable physics use:"
```

Comment:
This should on be printed if `${BULLET}` is set. Otherwise, you can't enable physics, right?

Reply:
Yes, the grab/release actions don't work when Bullet is not enabled

Reply:
@Ram81 In fact, it should probably print a warning to the effect it's not built with bullet.

Reply:
User will still be able to use the webGL setup in browser even when build without bullet. It will onlt disable grab release actions when not built with bullet. Should it still be a warning?

Reply:
I guess the lack of this message sort of acts as a warning...